packages/editor-ui/src/components/Projects/ProjectMoveResourceConfirmModal.vue

```diff
@@ -0,0 +1,94 @@
+<script lang="ts" setup>
+import { ref, computed } from 'vue';
+import type { ICredentialsResponse, IWorkflowDb } from '@/Interface';
+import { useI18n } from '@/composables/useI18n';
+import { useUIStore } from '@/stores/ui.store';
+import { useProjectsStore } from '@/stores/projects.store';
+import Modal from '@/components/Modal.vue';
+import { N8nCheckbox, N8nText } from 'n8n-design-system';
+import { useToast } from '@/composables/useToast';
+
+const props = defineProps<{
+	modalName: string;
+	data: {
+		resource: IWorkflowDb | ICredentialsResponse;
+		resourceType: 'workflow' | 'credential';
+		projectId: string;
+	};
+}>();
+
+const i18n = useI18n();
+const toast = useToast();
+const uiStore = useUIStore();
+const projectsStore = useProjectsStore();
+
+const checks = ref([false, false]);
+const allChecked = computed(() => checks.value.every(Boolean));
+
+const moveResourceLabel = computed(() =>
+	props.data.resourceType === 'workflow'
+		? i18n.baseText('projects.move.workflow.confirm.modal.label')
+		: i18n.baseText('projects.move.credential.confirm.modal.label'),
```

Comment:
Maybe `i18n.baseText(`projects.move.${props.data.resourceType}.confirm.modal.label`)` ?

Reply:
I intentionally avoid doing this because of a more straightforward lookup of the translation key

Reply:
I see. That's a good point.